README.md

```diff
@@ -40,6 +40,56 @@ To be able to use the GitHub API, you'll need to:
 - [Register a new developer application](https://github.com/settings/developers) in your profile.
 - Open [src/GitHub.App/Api/ApiClientConfiguration.cs](src/GitHub.App/Api/ApiClientConfiguration.cs) and fill out the clientId/clientSecret fields for your application.
 
+Build using Visual Studio 2015 or:
+
+```txt
+build.cmd
+```
+
+Install in live (non-Experimental) instances of Visual Studio 2015 and 2017:
+
+```txt
+install.cmd
+```
+
+Note, the script will only install in one instance of Visual Studio 2017 (Enterprise, Professional or Community).
+
+## Build Flavors
+
+By default, building will create a VSIX with `Experimental="true"` and `AllUsers="false"` in its `extension.vsixmanifest`. These settings are necessary in order to easily install a standalone VSIX file. There is no need to uninstall the version previously installed via Visual Studio setup / Extensions and Updates.
+
+To build and install a `Debug` configuration VSIX:
+```txt
+build.cmd
+install.cmd
+```
+
+To build and install a `Release` configuration VSIX:
+```txt
+set Configuration=Release
+build.cmd
+install.cmd
+```
+
+To build a VSIX that could be deployed via the gallery / Extensions and Updates:
+```txt
+set Configuration=Release
+set IsExperimental=false
+build.cmd
+install.cmd
+```
+
+To build a VSIX that could be deployed via Visual Studio setup:
+```txt
+set Configuration=Release
+set IsExperimental=false
+set IsProductComponent=false
```

Comment:
I wanted to keep everything together as much as possible. The Roslyn build uses similar stuff, so it isn't secret knowledge or anything. Since the .csproj files are public, I thought this documentation for them should be as well.